src/test/javascript/spec/service/programming-submission.service.spec.ts

```
@@ -110,8 +147,14 @@ describe('ProgrammingSubmissionService', () => {
     it('should query httpService endpoint and setup the websocket subscriptions if no subject is cached for the provided participation', () => {
         httpGetStub.mockReturnValue(of(currentSubmission));
         let submission;
+        submissionService.setLocalCIProfile(false);
         submissionService.getLatestPendingSubmissionByParticipationId(participationId, 10, true).subscribe((sub) => (submission = sub));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)